src/magick-image.ts

```diff
@@ -205,6 +205,8 @@ export interface IMagickImage extends IDisposable {
     modulate(brightness: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage): void;
     modulate(brightness: Percentage, saturation: Percentage, hue: Percentage): void;
+    motionBlur():void;
```

Comment:
Can you please remove this overload? The user will always need to specify the values. In ImageMagick the last value is not mandatory but I don't want to create a specific overload for this. And this overload is also not available in the Magick.NET api and I am trying to keep this library as close as possible to the Magick.NET library.